api/python/quilt3/data_transfer.py

```diff
@@ -876,7 +876,7 @@ def copy_file(src: PhysicalKey, dest: PhysicalKey, size=None, message=None, call
     """
     def sanity_check(rel_path):
         for part in rel_path.split('/'):
-            if part in ('', '.', '..'):
+            if part in ('.', '..'):
```

Comment:
is this really needed to have `set_dir()` working?
I don't think so, as I understand it's only needed for file uploading/copying
I think that change makes some sense for src (maybe we even shouldn't check src at all), but I'm not sure it's a good idea to allow empty path segments in dst
if we go with this change it needs to be tested and to be reflected in changelog and PR title
might make sense to move it to a separate PR

Reply:
Huh, I thought it was. Maybe that was a side effect of a malformed test. My bad.